platform/ui/src/views/StudyList/components/Header.js

```diff
@@ -1,10 +1,46 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
 
-/**
- * THIS FILE LOCATION IS TEMPORARY AND SHOULD NOT BE PLACED HERE
- */
-const Header = () => {
-  return <div>{`<- HEADER CONTENT ->`}</div>;
+import { IconButton, Icon } from '@ohif/ui';
+
+import OHIFLogo from './OHIFLogo.js';
+
+function Header({ appLogo = OHIFLogo(), children, t }) {
+  const showSettingsDropdown = () => {
+    // TODO: Update once dropdown component is created
+  };
+
+  return (
+    <div className="bg-custom-navy flex flex-row justify-between p-2 text-white">
+      <div className="flex items-center">
+        <div>{appLogo}</div>
+        <div>{children}</div>
```

Comment:
not sure if we need a `children`

Reply:
I have added this because we will need to display some links in header, as `Back to viewer` and `Study List` or anything else. 
In this case I will leave it to the viewer to control. Thats why I kept the children.

Reply:
Great!